src/adapters/simpleble.ts

```diff
@@ -85,10 +87,7 @@ export interface Peripheral {
 }
 
 /** SimpleBLE Adapter. */
-export interface Adapter {
-    identifier: string;
-    address: string;
-    active: boolean;
+export interface Adapter extends HardwareAdapterDetails {
```

Comment:
Is it worth merging these types?

Reply:
To have a clearly typed separateion between whats exposed as "information" and whats a full adapte rinstance I would leave it separated. Or what do you mean?

Reply:
If we change names I would rather rename HardwareAdapterDetails to just AdapterDetails ...

But yes this naming overlapping that we have "native BLE adapters" aka "Hardware level" and "BLE providers like Noble OR SimpleBLE" (at least structurewise) that are also called "Adapters is difficult.

I could imagine renaing Adapter -> provider fpr the SimpleBle/Noble level and keep Adapter for the "native level". WDYT?